packs/github/sensors/github_repository_sensor.py

```diff
@@ -42,6 +34,8 @@ def setup(self):
         if repository_sensor is None:
             raise ValueError('"repository_sensor" config value is required.')
 
+        self.EVENT_TYPE_WHITELIST = repository_sensor.get('event_type_whitelist', [])
```

Comment:
Minor thing - empty list doesn't really make sense so we can just throw if the list is empty (similar as the repositories check below).